README.md

```diff
@@ -66,7 +67,7 @@ const App = React.createClass({
 });
 
 
-React.render(<App />, document.body);
+ReactDOM.render(<App />, appRoot);
```

Comment:
That's intentional. These are README examples, not runnable examples. Or tutorials for React basics.

This variable just signifies rendering of the app into a dom node intended for the app, other than the body, which the developer should already have.